frontend/pages/home/components/HPL/AddSubaccount.tsx

```diff
@@ -148,7 +153,7 @@ const AddSubaccount = ({ setAssetOpen, open }: AddSubaccountProps) => {
                         return (
                           <button
                             key={k}
-                            className="p-1 flex flex-row justify-start items-center w-full gap-2 text-sm hover:bg-HoverColorLight dark:hover:bg-HoverColor"
+                            className="flex flex-row justify-start items-center w-full gap-2 text-sm hover:bg-HoverColorLight dark:hover:bg-[#33B2EF3D]"
```

Comment:
Same here. check color. should be registered. on future theme could be harder to do.